docs/pages/eas-update/request-proxying.mdx

```diff
@@ -0,0 +1,37 @@
+---
+title: Request Proxying
+description: Proxy requests to the EAS Update server through your own server.
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+EAS Update supports request proxying, which allows you to proxy requests to the EAS Update server through your own server. This can be useful for various reasons, such as adding custom headers, logging requests, or implementing additional security or request IP anonymization measures.
+
+## Enabling request proxying
+
+1. Create two proxy servers that will handle the requests:
+   - One for the update asset requests (JavaScript bundles, images, and so on).
+     - This must forward requests to `assets.eascdn.net`, the EAS Update asset server.
+     - This must pass-through all URL contents (path, query parameters, and so on).
+     - This must pass-through all headers prefixed with `expo-` or `eas-`.
+   - One for the update manifest requests.
+     - This must forward requests to `u.expo.dev`, the EAS Update server.
+     - This must pass-through all URL contents (path, query parameters, and so on).
+     - This must pass-through all headers prefixed with `expo-` or `eas-`.
+1. Add the following fields to your `eas.json` configuration file, replacing the placeholders with your actual proxy server URLs:
```

Comment:
```suggestion
1. Add the following fields to your **eas.json** configuration file, replacing the placeholders with your actual proxy server URLs:
```

As per our writing style guide, let's make the file name bold: https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#file-names-directory-names-file-extensions-as-bold-text